src/ui/components/AliasSelector.js

```diff
@@ -101,18 +101,21 @@ export function AliasSelector({
         <h2>Aliases:</h2>
       </label>
       <div>
-        {selectedItems.map((selectedItem, index) => (
-          <span
-            key={`selected-item-${index}`}
-            {...getSelectedItemProps({selectedItem, index})}
-          >
-            <Markdown
-              renderers={{paragraph: "span"}}
-              source={getNodeDescription(selectedItem)}
-            />
-            <br />
-          </span>
-        ))}
+        {selectedItems.map(
+          (selectedItem, index) =>
+            getNodeDescription(selectedItem) && (
+              <span
+                key={`selected-item-${index}`}
+                {...getSelectedItemProps({selectedItem, index})}
+              >
+                <Markdown
+                  renderers={{paragraph: "span"}}
+                  source={getNodeDescription(selectedItem)}
+                />
+                <br />
+              </span>
+            )
+        )}
```

Comment:
So.. basically it's an implicit filter that depends on the fact that react filters falsey-results, and presumably getNodeDescription is returning undefined for the aliases?

Too hard to reason about as a newcomer to the code imo. how about `selectedItems.filter(...).map`